### DISCUSSION TITLE
Texas Instrument TMS570 and RM4 support?

### DISCUSSION THREAD

**Discussion by spydatron** [modm-io/modm#931] [Ideas]
Wondering if there are any plans to support Cortex-R Microcontrollers like TMS570 and RM4 from Texas Instrument?

**Comment by salkinium** [comment]
At least not from my side, mainly due to lack of applications on my part. It's really difficult to properly support new devices I don't have at hand. Also unclear if there's a database of hardware descriptions available for these device families that could be added to [modm-io/modm-devices](https://github.com/modm-io/modm-devices) to help our code generator.
